### DISCUSSION TITLE
Why is the network interception of the rabbitmq server still going on, but the process of the client server has been killed, and there is no network connectio

### DISCUSSION THREAD

**Discussion by thesky2017** [rabbitmq/rabbitmq-server#6654] [Other]
Why is the network interception of the rabbitmq server still going on, but the process of the client server has been killed, and there is no network connection. You can also see the network connection in the background of the rabbitmq
rabbitmq server:
![image](https://user-images.githubusercontent.com/29980243/207204594-043da512-fe34-40c5-b1b8-12af21bfdd81.png)
client:
![image](https://user-images.githubusercontent.com/29980243/207204665-f9be821d-9ade-4bef-bfab-0010b6e04ff4.png)

**Comment by michaelklishin** [comment]
Because TCP connections do not close instantly and in multiple scenarios the server side of a connection can end up [in a special state](https://aws-labs.com/time-wait-state-tcp-effect-busy-servers/) that avoids two different clients connecting with the same client (ephemeral range) port.

We cannot suggest anything with two screenshots, one line each. See [RabbitMQ logs](https://www.rabbitmq.com/logging.html#logged-events) and TCP connection states for clues.
